[Bug] g.gui.gmodeler: defining relations fails

## Describe the bug

If I try to define a relation using the `Define relation` option from the menu and choose two features (data and module), nothing happens, raising the following error:

```python
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/wx/lib/ogl/canvas.py", line 276, in OnMouseEvent
    nearest_object.GetEventHandler().OnLeftClick(x, y, keys, attachment)
  File "/home/pesek/workspace/grass/dist.x86_64-pc-linux-gnu/gui/wxpython/gmodeler/canvas.py", line 150, in OnLeftClick
    dlg = ModelRelationDialog(parent=self.frame, shape=rel)
  File "/home/pesek/workspace/grass/dist.x86_64-pc-linux-gnu/gui/wxpython/gmodeler/dialogs.py", line 399, in __init__
    self._layout()
  File "/home/pesek/workspace/grass/dist.x86_64-pc-linux-gnu/gui/wxpython/gmodeler/dialogs.py", line 421, in _layout
    mainSizer.Add(
wx._core.wxAssertionError: C++ assertion "CheckSizerFlags(!((flags) & (wxALIGN_RIGHT | wxALIGN_CENTRE_HORIZONTAL | wxALIGN_BOTTOM | wxALIGN_CENTRE_VERTICAL)))" failed at ./src/common/sizer.cpp(2299) in DoInsert(): wxALIGN_RIGHT | wxALIGN_CENTRE_HORIZONTAL | wxALIGN_BOTTOM | wxALIGN_CENTRE_VERTICAL will be ignored in this sizer: wxEXPAND overrides alignment flags in box sizers

DO NOT PANIC !!

If you're an end user running a program not developed by you, please ignore this message, it is harmless, and please try reporting the problem to the program developers.

You may also set WXSUPPRESS_SIZER_FLAGS_CHECK environment variable to suppress all such checks when running this program.

If you're the developer, simply remove this flag from your code to avoid getting this message. You can also call wxSizerFlags::DisableConsistencyChecks() to globally disable all such checks, but this is strongly not recommended.
```

## Expected behavior

Pop up a dialog.

## System description

- Operating System: Linux (Ubuntu)
- GRASS GIS version: `main` as of ae7163c50e1d8f46b9c5871e3de96b973cf20f0f

--- a/gui/wxpython/gmodeler/dialogs.py
+++ b/gui/wxpython/gmodeler/dialogs.py
@@ -412,9 +412,7 @@ class ModelRelationDialog(wx.Dialog):
             flag=wx.EXPAND | wx.LEFT | wx.RIGHT | wx.BOTTOM,
             border=5,
         )
-        mainSizer.Add(
-            btnSizer, proportion=0, flag=wx.EXPAND | wx.ALL, border=5
-        )
+        mainSizer.Add(btnSizer, proportion=0, flag=wx.EXPAND | wx.ALL, border=5)
 
         self.panel.SetSizer(mainSizer)
         mainSizer.Fit(self)

--- a/gui/wxpython/gmodeler/dialogs.py
+++ b/gui/wxpython/gmodeler/dialogs.py
@@ -413,11 +413,11 @@ class ModelRelationDialog(wx.Dialog):
             border=5,
         )
         mainSizer.Add(
-            btnSizer, proportion=0, flag=wx.EXPAND | wx.ALL | wx.ALIGN_CENTER, border=5
+            btnSizer, proportion=0, flag=wx.EXPAND | wx.ALL, border=5
         )
 
         self.panel.SetSizer(mainSizer)
-        mainSizer.Fit(self.panel)
+        mainSizer.Fit(self)
 
         self.Layout()
         self.SetSize(self.GetBestSize())

